### PR TITLE
returns node name instead id if value is a udi

### LINKED ISSUE #5545: Feature - UDIs in list views should be converted to something more logical
When adding a custom property to a listview, In this case a node picker, the displayed text is the umbId - > umb://document/e853bf611ba243bfb23fa2105d119614. 

The error occurs on line 17978 in Umbraco.Controller.js.

```
if (typeof value === 'undefined') {
     value = getCustomPropertyValue(alias, result.properties);
 }
```

The function does its work, but that type of information (umbId) is errelevant for the user. 

REndering example : 
image

### PR DESCRIPTION
**Pull request by skartknet**
**Please note:** I'm not sure if the right way of getting the node name is using `node.variants[0].name` but it's the only way I found. This is in line 653.

### Prerequisites

- [x] I have added steps to test this contribution in the description below

this fixes #5545 

### Description
If the value of the custom property is a Udi, it'll get the node and return its name instead the raw value.

### How to test
Create a container doctype. Allow in root. Enable list view.
Create a DocType with a content picker. Allow this as child for the previous container.
Add the picker property to the list view of the container.
Create a couple of pages using that doctype. Configure the picker in one of them selecting the other one.
Navigate to the container's list view. You should get the name of the selected node instead its Id.

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.controller.js [hunk 1]
```diff
@@ -642,10 +644,23 @@ function listViewController($scope, $routeParams, $injector, $timeout, currentUs
         }
 
         if (foundAlias) {
+
             value = properties[index].value;
+
+            if (value.startsWith("umb://")) {
+                contentResource.getById(value).then(function (node) {
+                    if (node) {
+                        deferred.resolve(node.variants[0].name);
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.controller.js]: Maybe have a look at this PR and see if it's possible to do the same to get the node name with the current culture?

https://github.com/umbraco/Umbraco-CMS/pull/5408

Comment (skartknet) [reply src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.controller.js]: Thanks Sebastiaan. The code is different to that PR but I found the way to get the current culture.

**Comment by stherm** [comment]
Will this affect the sort as well. I've made this kind of fix in a custom listview controller. But the sort still use the umb id.

**Comment by skartknet** [comment]
Not sure where you mean, the sort of pages list only shows the page name and creation date.

**Comment by stherm** [comment]
@skartknet By default yes, the sort is only available on those column. But if you activate the sort on custom added field in the list view (like the content of a nodePicker), the sort uses the umbID and not the name.

**Comment by nul800sebastiaan** [comment]
Thanks @skartknet for the attempt, unfortunately I think @stherm has pointed out a fatal flaw in this one, the sorting doesn't update correctly. I'm afraid that if we want to fix this one it will have to be on the server side, but I'm not sure what the best way to do that would be, maybe @Shazwazza  can pitch in.

Please note that the other hurdle with this PR was that not all content is multilingual, so I had to add some added null checks as well (`var variant = node.variants.filter(v => v.language && v.language.culture && v.language.culture === $scope.options.cultureName)[0];`)

I'll close this one for now as it will require a complete rethink.

Thanks again for the efforts so far!

**Pull request closed without merging**